libs/api/v1/src/lib/api-v1.ts

```diff
@@ -23,8 +24,10 @@ export const apiV1 = new Elysia({ prefix: '/v1' })
       tags: ['v1'],
     },
   })
-  .onBeforeHandle(({ headers, error }) => {
-    validateApiKey(headers.authorization) && error(401, 'Provided API key is invalid.')
+  .onBeforeHandle(async ({ headers, error }) => {
+    const errorCode: ApiErrorCode | undefined = await validateApiKey(headers.authorization)
+    console.log('dump', errorCode)
```

Comment:
This is buggy. When I call request, it executes, it even shows an error, but it doesn't cause the request to terminate.

<img width="577" alt="Snímek obrazovky 2024-09-18 v 15 14 59" src="https://github.com/user-attachments/assets/ab10f102-7f43-41de-a491-7e0da7c90350">

<img width="795" alt="Snímek obrazovky 2024-09-18 v 15 15 08" src="https://github.com/user-attachments/assets/d6fede35-83a8-42d5-907d-bca259aef4a4">